doc/needles.txt

```diff
@@ -1,90 +0,0 @@
-Some packages are required to run crop.py. So for the first time do:
```

Comment:
IMHO this file should not be removed as is. Certainly, "openqa" references should be removed but it also describes assert_screen and such which are actually defined in os-autoinst (not openqa). Other files are fine to cleanup.